lib/v2/screens/explore_screens/plant_seeds/interactor/viewmodels/plant_seeds_state.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+/// --- STATE
+class PlantSeedsState extends Equatable {
+  final PageState pageState;
+  final RatesState ratesState;
+  final String? fiatAmount;
```

Comment:
instead if being nullable, 
Make them not nullable 

ANd on initial, initialize them to 0

Reply:
done